src/serac/infrastructure/CMakeLists.txt

```diff
@@ -41,9 +41,10 @@ set(infrastructure_sources
     terminator.cpp
     )
 
-set(infrastructure_depends axom::inlet axom::fmt axom::cli11 mfem ${serac_device_depends})
+set(infrastructure_depends axom::inlet axom::fmt axom::cli11 camp mfem ${serac_device_depends})
 blt_list_append(TO infrastructure_depends ELEMENTS tribol IF TRIBOL_FOUND)
 blt_list_append(TO infrastructure_depends ELEMENTS caliper adiak::adiak IF SERAC_ENABLE_PROFILING)
+blt_list_append(TO infrastructure_depends ELEMENTS blt::openmp IF SERAC_ENABLE_OPENMP)
```

Comment:
I don't think this is needed, but the openmp dependency for mfem wasn't picked up through `FindMFEM.cmake`.

Reply:
what happens when you call `blt_print_target_properties(TARGET mfem)`? also can you check to see if mfem was built with `+openmp` in the concretizer tree?